tests/MageTest/Command/AbstractCommandTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace MageTest\Command;
+use Mage\Command\AbstractCommand;
+use MageTest\TestHelper\BaseTest;
+use PHPUnit_Framework_MockObject_MockObject;
+
+/**
+ * Class AbstractCommandTest
+ * @package MageTest\Command
+ * @author Jakub Turek <ja@kubaturek.pl>
+ * @coversDefaultClass Mage\Command\AbstractCommand
+ */
+class AbstractCommandTest extends BaseTest
+{
+    /**
+     * @var AbstractCommand|PHPUnit_Framework_MockObject_MockObject
+     */
+    private $abstractCommand;
+
+    /**
+     * @before
+     */
+    public function before()
+    {
+        $this->abstractCommand = $this->getMockForAbstractClass('Mage\Command\AbstractCommand');
+    }
+
+    /**
+     * @covers ::setConfig
+     */
+    public function testSetConfig()
+    {
+        $configMock = $this->getMock('Mage\Config');
+        $this->abstractCommand->setConfig($configMock);
+
+        $actual = $this->getPropertyValue($this->abstractCommand, 'config');
+        $this->assertEquals($configMock, $actual);
+    }
+
+    /**
+     * @covers ::getConfig
+     */
+    public function testGetConfig()
```

Comment:
There is no refactoring needed for using depends-annotation. When used, one of the tests doesn't need Reflection anymore but depends on the positive test result of the other one. I think the less we are using Reflection functionality together with the tests, the more stable it gets, because we don't "test" Reflection with it.